components/org.wso2.carbon.security.caas.userstore.filebased/src/main/java/org/wso2/carbon/security/caas/userstore/filebased/connector/FileBasedIdentityStoreConnector.java

```diff
@@ -70,7 +71,7 @@ public void init(IdentityStoreConnectorConfig identityStoreConnectorConfig)
             throw new IdentityStoreException("storeFile property is not provided for file based connector");
         }
 
-        userStorePath = Paths.get(userStoreFile);
+        userStorePath = Paths.get(CarbonSecurityConstants.getCarbonHomeDirectory().toString(), userStoreFile);
```

Comment:
This is not right. The user store file path is retrieved from configs and carbon home is not needed for that.